release-notes/10.0/preview/preview7/dotnetmaui.md

```diff
@@ -2,12 +2,181 @@
+- **TabbedPageManager**: Made TabbedPageManager public for advanced customization scenarios
+- **StackNavigationManager**: Exposed public APIs for StackNavigationManager on Android
+
+## Deprecated API Removals
+
+As part of .NET 10, several deprecated APIs have been removed:
+
+- **Accelerator class**: Removed from Microsoft.Maui.Controls
+- **ClickGestureRecognizer**: Removed in favor of TapGestureRecognizer
+- **Page.IsBusy**: Marked as obsolete
+
+## Control and Layout Improvements
+
+This release includes numerous fixes and improvements across controls and layouts:
+
+### ScrollView Improvements
+
+- Fixed Android ScrollView content measurement issues
+- Resolved iOS extra bottom space when using SafeAreaEdges
+- Fixed ScrollView ContentOffset reset on first layout pass
+
+### Image and Visual Improvements
+
+- Fixed random image disappearing when switching tabs on Android
+- Improved image resizing with proper disposal when `disposeOriginal` is set to true
+- Enhanced FontImageSource sizing for back/flyout icons on iOS
+
+### Navigation and Shell Fixes
+
+- Fixed Shell tab becoming blank after specific navigation patterns on iOS
+- Improved handler disconnection when removing non-visible pages from navigation stack
+- Fixed Shell custom FlyoutIcon display problems on iOS
+
+### Platform-Specific Fixes
+
+- **Android**: Fixed Picker selected item highlight, resolved OnIdiom threading issues
+- **iOS**: Fixed Switch OffColor persistence, improved SwipeView content change handling
+- **Windows**: Enhanced WebView behavior with hardware acceleration disabled
+
+### Performance and Memory
+
+- Removed Compatibility.Layout dependency on modern .NET MAUI controls
+- Improved handler mapping efficiency by skipping useless calls
+- Enhanced nullability annotations for converter classes
+
+## .NET for Android
+
+This release includes continued improvements to the CoreCLR runtime integration, build tooling, and native interop performance:
+
+- **CoreCLR Runtime Enhancements**:
+  - Implemented GC bridge support for better memory management integration
+  - Added support for ignoring specific assemblies when requested to load them
+  - Improved native typemap code generation with fewer relocations
+  - Static linking of app native runtime/library for better performance
+
+- **Build and Tooling Improvements**:
+  - Updated p/invoke tables for `libSystem.Native.so`
+  - Enhanced logging with function names in native code
+  - Cleaned up workload definitions by removing deprecated android-net8 workload
+  - Fixed build warnings and improved project configuration
+
+- **Configuration Updates**:
+  - Set default for Http3Support feature switch
+  - Added `DOTNET_SYSTEM_NET_SECURITY_NOREVOCATIONCHECKBYDEFAULT` environment variable
+  - Removed legacy dotnet9 NuGet feeds
+
+- **Developer Experience**:
+  - Removed `IsRunningOnDesktop` checks for Android designer scenarios
+  - Updated TSA Area Path configuration for better tooling integration
```

Comment:
So, when I look at the diff we really did everything towards making CoreCLR run on Android, but we don't want to highlight in release notes yet.

There are useful fixes it doesn't cover:
* https://github.com/dotnet/android/commit/4fe5d419132c1f7c03cc600f096060ccb050db34
* https://github.com/dotnet/android/commit/6c2ac386100630ab7685b91a07ae82b263eabd4d

```suggestion
This release includes continued integration with multiple .NET runtimes, and several bug fixes:

- Fixes `=` symbol in environment variables.
- Fixes Java libraries to be redistributed in NuGet packages.
```
I'll work on a bigger list here in a few.